commands/util/charts.js

```diff
@@ -1,28 +1,151 @@
 import { SlashCommandBuilder, EmbedBuilder } from 'discord.js';
+import axios from 'axios';
```

Comment:
Why are we using axios here? The rest of the bot uses the built in fetch function so why add this now?